naas/requests/email_notification_basics.py

```diff
@@ -0,0 +1,65 @@
+from naas.client import Client
+
+
+class EmailNotificationBasics:
+
+    @classmethod
+    def create(cls, params=None):
+        """
+        Create a new record
+
+        :param params: dict
+        :return: Response
+        """
+        if params is None:
+            params = {}
+        request_body = {
+            "email_notification_basic": params
+        }
+        rel = Client.rel_for('rels/email-notification-basic')
+        route = Client.routes().route_for(rel)
+        url = route.url_for()
+        request = Client.post(url, data=request_body)
+        return request
```

Comment:
@arthuroe - `EmailNotificationBasic.create('nate@theklaibers.com', 'project-name', 'campaign-name', 'template-name')` will give me the `Configuration` error, but when I inspect what is being sent to the API I see that it's not sending all of the attributes over. It's sending a hash that looks like this:

```
Started POST "/email-notifications/basic" for 75.179.34.110 at 2019-05-03 15:00:30 -0400
Processing by API::V1::EmailNotificationsBasicController#create as NAAS_JSON
  Parameters: {"email_notification_basic"=>"tags"}
```

As you can see, there are no other attributes in there.